ci/openshift-ci/peer-pods-azure.sh

```diff
@@ -10,6 +10,8 @@
 #          resource group in "eastus" region and peers the network. You
 #          have to remove these manually (or use temporary accounts)
 
+SCRIPT_DIR=$(dirname "$0")
```

Comment:
Just curious what is the context where you expect to run it from elsewhere? Is it worth adding to the cover letter?

Reply:
this is actually inspired by me testing it on my computer. Also bisecting is done on computer and not in CI so from time to time this script needs to be executed by users (ok, let's face it, by me and I tend to forget where it needs to be executed from)

Reply:
Understood. Thanks, good idea.